test-data/server.key

```diff
@@ -0,0 +1,27 @@
+-----BEGIN RSA PRIVATE KEY-----
```

Comment:
You don't need to keep this in a file you can just directly keep this in file as a global []byte array, avoids directories like this.